sdk/core/azure-core/tests/test_streaming.py

```diff
@@ -165,6 +166,22 @@ def test_decompress_compressed_header(http_request):
     decoded = content.decode('utf-8')
     assert decoded == "test"
 
+
+@pytest.mark.parametrize("http_request", HTTP_REQUESTS)
+def test_decompress_compressed_header_offline(port, http_request):
+    client = PipelineClient("")
+    request = http_request(method="GET", url="http://localhost:{}/streams/decompress_header".format(port))
+    with RequestsTransport() as sender:
+        try:
+            response = client._pipeline.run(request, stream=True).http_response
+            response.raise_for_status()
+            data = response.stream_download(sender, decompress=True)
+            content = b"".join(list(data))
+            decoded = content.decode('utf-8')
+            assert decoded == "test"
+        except HttpResponseError as e:
```

Comment:
we can remove the try/except debugging code now that we fixed the issue with the gzip file.

Reply:
@zhzhh97 sorry if this was unclear -- I meant the try/except from the test you are adding here. It looks like the try/except from a different test might have accidentally been removed at the end of this file.